e2e/sbin/test.sh

```diff
@@ -1,32 +1,41 @@
 #!/bin/bash
 
-# Define constants
-HOST=localhost
-L1_PORT=8545
-L2_PORT=4011
-
-# Define directory structure
+# Configure variables
 SBIN_DIR=`dirname $0`
 SBIN_DIR="`cd "$SBIN_DIR"; pwd`"
-PROJECT_DIR=$SBIN_DIR/../project
-PROJECT_DATA_DIR=$PROJECT_DIR/specular-datadir
-CONTRACTS_DIR=$SBIN_DIR/../../contracts
-DOCKER_DIR=$SBIN_DIR/../../docker
-
-# Spin up containers
-cd $PROJECT_DIR
-docker compose -f $DOCKER_DIR/docker-compose-integration-test.yml up -d
-sleep 30
-$SBIN_DIR/wait-for-it.sh -t 60 $HOST:$L1_PORT
-$SBIN_DIR/wait-for-it.sh -t 60 $HOST:$L2_PORT
+set -o allexport
+source $SBIN_DIR/configure.sh
+set +o allexport
+
+# Spin up L1 node
+cd $CONTRACTS_DIR
+anvil --block-time 1 > $PROJECT_LOG_DIR/l1.log 2>&1 &
+ANVIL_PID=$!
+npx hardhat deploy --network localhost
+
+# Spin up L2 node
+cd $PROJECT_DATA_DIR
+$SBIN_DIR/sequencer.sh > $PROJECT_LOG_DIR/l2.log 2>&1 &
+L2GETH_PID=$!
```

Comment:
should we run a validator as well? even if we don't explicitly test it, it would at least make sure the happy path is working.